the-blue-alliance-ios/View Controllers/Events/EventsStoryboard.storyboard

```diff
@@ -439,6 +439,24 @@
             </objects>
             <point key="canvasLocation" x="3766" y="-41"/>
         </scene>
+        <!--Event Rankings Table View Controller-->
+        <scene sceneID="TV6-tf-E65">
+            <objects>
+                <tableViewController id="CvT-u0-qAP" customClass="EventRankingsTableViewController" customModule="the_blue_alliance_ios" customModuleProvider="target" sceneMemberID="viewController">
+                    <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" dataMode="prototypes" style="plain" separatorStyle="default" rowHeight="44" sectionHeaderHeight="28" sectionFooterHeight="28" id="qSk-IJ-lzD">
+                        <rect key="frame" x="0.0" y="0.0" width="375" height="559"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <color key="backgroundColor" white="1" alpha="1" colorSpace="calibratedWhite"/>
+                        <connections>
+                            <outlet property="dataSource" destination="CvT-u0-qAP" id="ncW-7S-VdT"/>
+                            <outlet property="delegate" destination="CvT-u0-qAP" id="qpN-dc-gaT"/>
+                        </connections>
+                    </tableView>
+                </tableViewController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="RZI-DR-gwW" userLabel="First Responder" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="2134" y="-798"/>
```

Comment:
Super picky but... can we move the event ranking table view controller down underneath the event view controller in the storyboard, with the reset of the root view segues? That way they're organized the same.